tokio/src/blocking.rs

```diff
@@ -0,0 +1,41 @@
+cfg_rt_core! {
+    pub(crate) use crate::runtime::spawn_blocking;
+    pub(crate) use crate::task::JoinHandle;
+}
+
+cfg_not_rt_core! {
+    use std::fmt;
```

Comment:
This pattern will mark `JoinHandle` and `spawn_blocking` with the "this is only supported on `rt-core`", even though it technically exists without it. This is probably ok in this case, but we should be aware of it.

Note that `JoinHandle` has been marked `rt-core` or `rt-util`.

Reply:
Yeah... we probably should collapse `rt-util` w/ `rt-core` tbh... I will attack this separately.

Reply:
I opened #2942 to track.